src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -136,7 +139,7 @@ public function load( $post = null ) {
 			return tribe_error( 'core:aggregator:invalid-record-object', [], [ $post ] );
 		}
 
-		if ( $post->post_type !== Tribe__Events__Aggregator__Records::$post_type ) {
+		if ( $post->post_type !== Records::$post_type ) {
```

Comment:
:no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: 805e217b143c188d633a2a340e2d3ea8</sub>